### DISCUSSION TITLE
挂载阿里云盘，使用infuse透过alist的webdav访问速率低下。

### DISCUSSION THREAD

**Discussion by zpljd258** [AlistGo/alist#1213] [Q&A] [answered]
<img width="603" alt="image" src="https://user-images.githubusercontent.com/11162658/173017360-dfa451e2-196e-45fb-a6cb-8f9100fc1e6f.png">
Hi there，

我的alist版本v2.6.1-0-geb15bce，挂载阿里云盘不管webdav direct功能是否打开，访问速度均为不到20Mbps，并且看起来是由最早的满速一直降低到30且在近期降低到不足20Mbps。

同样一台机器挂在迅雷云盘可以达到80Mbps左右的稳定速度。本身外网速率300Mbps，确认过自身网络环境内效能良好。请问是否有朋友知道如何解决这个问题？？

**Comment by zpljd258** [comment]
BTW，网盘账号/alist没有分享给他人使用。均为个人看电影使用。应该不会触发阿里云限速策略才对

**Comment by zpljd258** [comment, marked as answer]
这边贴上解决方案也希望后面遇到类似问题的朋友减少折腾的时间：

经查是因为surge的规则不匹配导致的，请加入bugsnag.com/flurry.com域名走国内链接即可解决问题。

上述两个域名加入后，速度由不到20Mbps提高到180Mbps左右，满足了正常4k视频的传输需求。如果使用迅雷云盘遇到速度问题，请将自己的nas域名加入科学软件走直链，此时迅雷云盘可以达到80Mbps。

**Reply by sheldonl3** [reply]
请问能否详细说一下方法步骤呢？

**Reply by zpljd258** [reply]
> 请问能否详细说一下方法步骤呢？

我认为步骤说的很详细了，建议您在认真阅读。

主要是讲我描述的问题加入到您的科学软件的规则列表里。

若您的确对此并不了解，我的建议是您也可以在需要使用infuse观看时关闭科学软件

**Reply by Dashan-IZ** [reply]
求大佬告知。v3版本怎么设置webdav啊，搞了半天不会，都失败，同软件

**Reply by Brainiac5000** [reply]
我没用surge也有这个问题，我真的纳闷了。alist 挂载115在apple tv的infuse读，20mbps，我直接傻了。同局域网的windows机子速度就是正常的。

**Comment by Dashan-IZ** [comment]
Brainiac5000 ***@***.***>于2022年12月15日 周四17:11写道：

> 我没用surge也有这个问题，我真的纳闷了。alist 挂载115在apple
> tv的infuse读，20mbps，我直接傻了。同局域网的windows机子速度就是正常的。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/1213#discussioncomment-4408169>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJAAGZE7FCZU4532CBLWNLOFHANCNFSM5YMY7KSQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
我的好像解决了、现在用infuse、速度挺快的

**Reply by Brainiac5000** [reply]
厉害，怎么做到，学一下，感谢。比较有意思的是我用clouddrive挂载infuse虽然不能达到满速，但是也能跑到200 mbps。我感觉是infuse的机制和alist正好不对付造成的。因为内网的windows速度和搭载服务器的本机是几乎一样。apple tv无论是有线还是无线接入都是慢速20mbps。

**Comment by Dashan-IZ** [comment]
Brainiac5000 ***@***.***>于2022年12月15日 周四17:47写道：

> 厉害，怎么做到，学一下，感谢。比较有意思的是我用clouddrive挂载infuse虽然不能达到满速，但是也能跑到200
> mbps。我感觉是infuse的机制和alist正好不对付造成的。因为内网的windows速度和搭载服务器的本机是几乎一样。apple
> tv无论是有线还是无线接入都是慢速20mbps。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/1213#discussioncomment-4408458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJCID3XINDMPJF6DTETWNLSLHANCNFSM5YMY7KSQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
我都是默认就好了、路径记得加/dav、如果你的alist部署用了nginx反向代理的话就是域名+80或者是443，就不是5244端口了、注意区分http和https就好了、其他倒是没什么需要注意的

**Reply by Brainiac5000** [reply]
我也是默认的设置没有用代理。你使用路由或者nas启动的alist吗，我使用windows做的。我继续去调看看哈。感谢

**Comment by Dashan-IZ** [comment]
Brainiac5000 ***@***.***>于2022年12月15日 周四17:47写道：

> 厉害，怎么做到，学一下，感谢。比较有意思的是我用clouddrive挂载infuse虽然不能达到满速，但是也能跑到200
> mbps。我感觉是infuse的机制和alist正好不对付造成的。因为内网的windows速度和搭载服务器的本机是几乎一样。apple
> tv无论是有线还是无线接入都是慢速20mbps。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/1213#discussioncomment-4408458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJCID3XINDMPJF6DTETWNLSLHANCNFSM5YMY7KSQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
我的一下就链接上了、播放视频也是非常快

**Comment by Dashan-IZ** [comment]
Brainiac5000 ***@***.***>于2022年12月15日 周四17:47写道：

> 厉害，怎么做到，学一下，感谢。比较有意思的是我用clouddrive挂载infuse虽然不能达到满速，但是也能跑到200
> mbps。我感觉是infuse的机制和alist正好不对付造成的。因为内网的windows速度和搭载服务器的本机是几乎一样。apple
> tv无论是有线还是无线接入都是慢速20mbps。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/1213#discussioncomment-4408458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJCID3XINDMPJF6DTETWNLSLHANCNFSM5YMY7KSQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
我用alist挂载的阿里云盘、代理默认302重定向就好了、速度非常快

**Comment by Dashan-IZ** [comment]
Brainiac5000 ***@***.***>于2022年12月15日 周四18:16写道：

> 我也是默认的设置没有用代理。你使用路由或者nas启动的alist吗，我使用windows做的。我继续去调看看哈。感谢
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/1213#discussioncomment-4408689>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJFNYPBZ6VJG2Q6WNSLWNLVZFANCNFSM5YMY7KSQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
哈哈哈、我用的是服务器、如果你有openwrt的路由器也是可以用的，不过没有公网的话就只能局域网中访问

**Comment by Dashan-IZ** [comment]
零点 ***@***.***>于2022年12月15日 周四18:18写道：

>
>
> Brainiac5000 ***@***.***>于2022年12月15日 周四18:16写道：
>
>> 我也是默认的设置没有用代理。你使用路由或者nas启动的alist吗，我使用windows做的。我继续去调看看哈。感谢
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/alist-org/alist/discussions/1213#discussioncomment-4408689>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASUBRJFNYPBZ6VJG2Q6WNSLWNLVZFANCNFSM5YMY7KSQ>
>> .
>> You are receiving this because you commented.Message ID:
>> ***@***.***>
>>
> 哈哈哈、我用的是服务器、如果你有openwrt的路由器也是可以用的，不过没有公网的话就只能局域网中访问

你可以试试我的webdav链接看看、http协议、地址specialcloud.top，账号webdav、密码webdav、高级选项里路径填/dav、端口80、然后保存、链接成功后、在现实的文件夹后边点亮星星收藏就好了

**Reply by Brainiac5000** [reply]
行，我尝试一下代理

**Comment by adeb** [comment]
我试了楼主的方法，不行，然后用电脑 ping www.aliyun.com，把识别到的aliyun IP地址添加到梯子里的直连规则里就OK了。

**Reply by CooperHash** [reply]
但应该还不止www.aliyun.com，播放的api host应该也要添加

**Reply by boobb** [reply]
能不能请教一下，如果本身是用alist挂载的夸克，应该如何获取播放的api和host，刚刚开始玩冒昧求教
